pages/api/courses/mentors/[mentor_id].js

```diff
@@ -1,6 +1,27 @@
 // GET: /pages/api/courses/mentors/[mentor_id].js
 
-export default function getCoursesByMentorId(req, res) {
+import { db } from '../../../../utils/api/firebase.config';
+import {
+  collection,
+  doc,
+  getDoc,
+  getDocs,
+  addDoc,
+  query,
+  where
+} from "firebase/firestore";
+
+export default async function getCoursesByMentorId(req, res) {
   const { mentor_id } = req.query;
-  // open to add query
+  try {
+    const q = query(collection(db, 'courses'), where('mentor.id', '==', Number(mentor_id)));
```

Comment:
Same as earlier comment. Only use single letter variable names in select situations